basics/5 terminal/index.md

```diff
@@ -1,27 +1,147 @@
-# Getting started with Git and the terminal
+# Getting started with Git and the Terminal
```

Comment:
Markdown: iets prettiger de source bewerken als er een witregel staat na elk kopje. Probeer `<br>` te vermijden, eventueel kunnen we kijken of de CSS van de site beter kan.

Reply:
De witregels kan ik toevoegen, de `<br>`'s zijn er vanwege de verschillende paddings rondom afbeeldingen. Ik ga dat fixen door de afbeeldingen te vervangen.